cohorts/cohort.py

```diff
@@ -576,9 +573,6 @@ def _load_single_patient_variants(self, patient, filter_fn, use_cache=True, **kw
             filter_fn_name = self._get_function_name(filter_fn)
             logger.debug("loading variants for patient {} with filter_fn {}".format(patient.id, filter_fn_name))
             use_filtered_cache = use_cache
-            if sys.version_info < (3, 3):
```

Comment:
Given our requirement is for python >=3 we should probably still check for versions 3.1 and 3.2?